File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/OldManual.java
package org.rexcellentgames.burningknight.entity.item.accessory.equippable;

public class OldManual extends Equippable {
	@Override
	public void onEquip(boolean load) {
		super.onEquip(load);
		this.owner.modifyStat("fire_on_reload", 1f);
	}

	@Override
	public void onUnequip(boolean load) {
		super.onUnequip(load);
		this.owner.modifyStat("fire_on_reload", -1f);
	}
}

File: core/src/org/rexcellentgames/burningknight/entity/item/accessory/equippable/OldManual.kt
package org.rexcellentgames.burningknight.entity.item.accessory.equippable

class OldManual : Equippable() {
	override fun onEquip(load: Boolean) {
		super.onEquip(load)
		this.owner.modifyStat("fire_on_reload", 1f)
	}

	override fun onUnequip(load: Boolean) {
		super.onUnequip(load)
		this.owner.modifyStat("fire_on_reload", -1f)
	}

	override fun canBeUpgraded(): Boolean {
		return false
	}
}